ads/google/adsense.js

```diff
@@ -39,22 +40,23 @@ export function adsense(global, data) {
   s.src = 'https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js';
   global.document.body.appendChild(s);
 
-  const i = global.document.createElement('ins');
-  i.setAttribute('data-ad-client', data['adClient']);
+  const attributes = Object.create(null);
+  attributes['data-ad-client'] = data['adClient'];
   if (data['adSlot']) {
-    i.setAttribute('data-ad-slot', data['adSlot']);
+    attributes['data-ad-slot'] = data['adSlot'];
   }
   if (data['adHost']) {
-    i.setAttribute('data-ad-host', data['adHost']);
+    attributes['data-ad-host'] = data['adHost'];
   }
   if (data['adtest'] != null) {
-    i.setAttribute('data-adtest', data['adtest']);
+    attributes['data-adtest'] = data['adtest'];
   }
   if (data['tagOrigin']) {
-    i.setAttribute('data-tag-origin', data['tagOrigin']);
+    attributes['data-tag-origin'] = data['tagOrigin'];
```

Comment:
this is a very common code pattern across 3Ps

to compact the code, can we introduce a function `removeUndefined()`, and do 
```
const attributes = removeUndefined({
  ...
  'data-tag-origin': data['tagOrigin'],
  ...
});
```